src/helpers/SchemeHelper.ts

```diff
@@ -0,0 +1,42 @@
+import { CardScheme, CustomScheme, RegularScheme, FibonacciScheme } from '../models/CardScheme';
+export default class SchemeHelper{
+
+     createScheme(schemeType: string, customScheme?: (string|number)[]): CardScheme | never {
+        let cardScheme: CardScheme = {type: '', scheme: []}
+        switch(schemeType) {
+            case 'fibonacci':
+                    cardScheme = this.createFibonacciScheme()
+                break;
+            case 'regular':
+                    cardScheme = this.createRegular()
+                break;
+            case 'custom':
+                    cardScheme = this.createCustom(customScheme || [])
+                break;
+             default:
+                throw new Error("not a valid card scheme type");
+        }
+        
+        return cardScheme;
```

Comment:
Het is in dit geval denk ik duidelijker en eenvoudiger om direct vanuit de `switch` `case`s te `return`en. Dit zorgt ervoor dat `cardScheme` en de `break` statements niet meer nodig zijn.